packages/package.go

```diff
@@ -76,7 +88,9 @@ func LoadPackageFromPath(path string) (*Package, error) {
 	if err != nil {
 		return nil, err
 	}
-	pkg := &Package{}
+	pkg := &Package{
+		WorkingDir: "/workdir",
```

Comment:
I have reworked the code for the field name to match docker container config structure and the configuration to match docker-compose.yml name